doc/developer/link-state.rst

```diff
@@ -198,8 +327,8 @@ Figure 1 below, illustrates the ZAPI Opaque message exchange between a
 message sequences are as follows:
 
 - First, both *Producer* and *Consumer* must register to their respective ZAPI
-  Opaque Message. **Link State Sync** for the *Producer* in order to receive
-  Database synchronisation request from a *Consumer*. **Link State Update** for
+  Opaque Message: **Link State Sync** for the *Producer* in order to receive
+  Database synchronisation request from a *Consumer*, **Link State Update** for
```

Comment:
If the consumer sends a Multicast msg requesting Link State Sync then the producer in order to send an Unicast response do a  zclient_opaque_decode but as far as no ZAPI_OPAQUE_FLAG_UNICAST flag no destination info (proto, instance ,session) if obtained so no way to do a Unicast to the proper consumer , isn't it?
Could you confirm that or explain the way to do properly in the consumer side.


This is my consumer side snippet
```
request.session_id = zclient->session_id
request.instance = zclient->instance
request.proto = zclient->redist_default
if((status = zclient_send_opaque(zclient, LINK_STATE_SYNC, (uint8_t *)&request, sizeof(request)))
```